<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaleWindow</class>
 <widget class="QDialog" name="SaleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Sale.</string>
  </property>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>511</height>
    </rect>
   </property>
   <property name="title">
    <string>Sale Information</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description :</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>211</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Items Available</string>
    </property>
    <widget class="QPushButton" name="addItemButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Item</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Qty:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="addQuantityComboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="availableItems">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="seller">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="customerLastName">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dateTimeLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>_________</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Transaction </string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="typeComboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Code:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="codeComboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dept Code :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="deptCodeComboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer Last Name :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="customerFirstName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="description">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date/Time :</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>211</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Items Sold </string>
    </property>
    <widget class="QListWidget" name="itemsSold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="removeQuantityComboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="removeItemButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Item</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Qty:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer First Name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sold By:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>480</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Price : </string>
    </property>
   </widget>
   <widget class="QLabel" name="runningTotalLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>480</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>$ 0.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected Item Price : </string>
    </property>
   </widget>
   <widget class="QLabel" name="priceInfoLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="finishOrderButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>550</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish Order</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>seller</tabstop>
  <tabstop>description</tabstop>
  <tabstop>customerFirstName</tabstop>
  <tabstop>customerLastName</tabstop>
  <tabstop>typeComboBox</tabstop>
  <tabstop>codeComboBox</tabstop>
  <tabstop>deptCodeComboBox</tabstop>
  <tabstop>availableItems</tabstop>
  <tabstop>itemsSold</tabstop>
  <tabstop>addItemButton</tabstop>
  <tabstop>addQuantityComboBox</tabstop>
  <tabstop>removeItemButton</tabstop>
  <tabstop>removeQuantityComboBox</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>finishOrderButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
